jwst/ramp_fitting/ramp_fit.py

```diff
@@ -1559,10 +1628,9 @@ def fit_next_segment(start, end_st, end_heads, pixel_done, data_sect, mask_2d,
         end_heads[end_heads < 0.] = 0.
 
     return f_max_seg, num_seg
-    # end of fit_next_segment()
 
 
-def fit_lines(data, mask_2d, rn_sect, gain_sect, ngroups, weighting, group_time):
+def fit_lines(data, mask_2d, rn_sect, gain_sect, ngroups, weighting):
```

Comment:
The doc string still defines `group_time`, but it has been removed from the calling sequence.